### PR TITLE
[plugin.video.dailymotion_com] 2.3.3

### PR DESCRIPTION
**Pull request by Gujal00**
### Description

The current maintainer L0RE has moved on to develop a different addon and this codebase has been left in limbo. As per L0RE, he is happy to handover maintenance of this addon to me.
https://forum.kodi.tv/showthread.php?tid=142626&pid=2656751#pid2656751
The changes include the following:
Make videos playable in Krypton
Cleanup LiveTV section to show only active streams.
Change artwork to reflect current Dailymotion branding
Remove all deprecated options

### Checklist:

- [X] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [X] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-plugins/blob/master/CONTRIBUTING.md) document
- [X] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practice but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-plugins/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.

### CODE REVIEW

**Comment by enen92** [comment]
Awesome. Many thanks for taking over this plugin, it is often requested on the forums. If you know of any addons that are not being maintained anymore (and are in fact broken) please let us know so they can be marked as broken

File: plugin.video.dailymotion_com/addon.xml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<addon id="plugin.video.dailymotion_com" name="DailyMotion.com" version="2.2.3" provider-name="L0RE">
+<addon id="plugin.video.dailymotion_com" name="DailyMotion.com" version="2.3.3" provider-name="AddonScriptorDE">
```
Comment (Razzeee) [inline plugin.video.dailymotion_com/addon.xml]: Shouldn't this be your name?

File: plugin.video.dailymotion_com/addon.xml [hunk 2]
```diff
@@ -10,11 +10,16 @@
...
+        <description lang="en">Dailymotion attracts 300 million users from around the world, who watch 3.5 billion videos on its player each month.</description>
         <disclaimer lang="en">Some parts of this addon may not be legal in your country of residence - please check with your local laws before installing.</disclaimer>
-        <website>http://www.dailymotion.com</website>
+		<summary lang="fr">Films, Séries TV, Musique, Clips, et Live</summary>
+		<language></language>
+		<description lang="fr">Dailymotion attire 300 millions d’utilisateurs qui regardent 3,5 milliards de vidéos sur son player chaque mois, à travers le monde.</description>
+		<disclaimer lang="fr">Certaines parties de cet addon peuvent ne pas être légales dans votre pays de résidence - s’il vous plaît vérifier avec vos lois locales avant d’installer.</disclaimer>
+        <news>Additional coding by alibaba011, gates, gujal, L0RE</news>
+		<website>http://www.dailymotion.com</website>
         <source>https://github.com/AddonScriptorDE/plugin.video.dailymotion_com</source>
```
Comment (Razzeee) [inline plugin.video.dailymotion_com/addon.xml]: Might make sense to have your repo here now?

Comment (Gujal00) [reply plugin.video.dailymotion_com/addon.xml]: What is the status of this PR, is it rejected?
Regarding your questions
1. Provider name. I left it as the original developer name and added mine under <news>. Please let me know if it is not acceptable
2. Fanart. Currently it is qHD size. Will try to find a bigger one, if not you can remove the fanart. The addon never had fanart to start with.

Comment (Razzeee) [reply plugin.video.dailymotion_com/addon.xml]: @Gujal00 
Please send a new PR with changes to 1, 2 and 3.  It needs to have your nick, if your going to take over development :)
If anything is unclear, let us know.

Comment (Gujal00) [reply plugin.video.dailymotion_com/addon.xml]: Hi thanks for that. So just a new PR with the above changes only (that is addon.xml and fanart.jpg) as all the other changes seem to have been merged

**Pull request merged**